ssg/templates.py

```diff
@@ -108,10 +110,10 @@ def __init__(
         self.output_dirs = dict()
         for lang in languages:
             lang_dir = lang
-            if lang == "oval":
-                # OVAL checks need to be put to a different directory because
-                # they are processed differently than remediations later in
-                # the build process
+            if lang == "oval" or lang.startswith("sce-"):
+                # OVAL and SCE checks need to be put to a different directory
```

Comment:
Why isn't the directory called `sec-bash`?

Reply:
The rule-dir SCE content is placed in a directory called `sce` with filename `{{{ rule_id }}}` and file extension of the language used (presumably -- we preserve whatever file extension was already in the rule_dir file -- e.g., `.py`, `.sh` &c). In the templating system, we lack knowledge of file extension given by the author (they're all `.template`) and so we need another mechanism to indicate file suffix. However, we still (for XCCDF and DS path) want to put it in the `sce/` directory. For now, Canonical only needs Bash content, so we've called it `sce-bash`. If someone else wanted to add, e.g., Python support, they _only_ need to do so in the templating system (adding `sce-python` and the correct `.py` suffix) and then the rest of the SCE infrastructure gets reused. 

HTH clarifies things.